packages/flutter_tools/test/integration.shard/isolated/native_assets_test_utils.dart

```diff
@@ -103,6 +117,21 @@ import '${packageName}_bindings_generated.dart' as bindings;
   await dartFile.writeAsString(dartFileNew2);
 }
 
+Map<String, Object?> _pubspecAsMutableJson(String pubspecContent) {
+  return json.decode(json.encode(loadYaml(pubspecContent))) as Map<String, Object?>;
+}
+
+void _updateDependencies(Map<String, Object?>? to, Map<String, Object?> from) {
```

Comment:
nit: `to` is typed as nullable here, but we always pass non-null values into it.

Reply:
Thanks (yes, I added bang operator on call sites, as this is guaranteed to be non nullable)

Done.